client-sdk/ts-web/core/reflect-go/main.go

```diff
@@ -704,7 +698,6 @@ func main() {
 	visitClientWithPrefix(&internal, reflect.TypeOf((*workerStorage.StorageWorker)(nil)).Elem(), "StorageWorker")
 	visitClient(&internal, reflect.TypeOf((*runtimeClient.RuntimeClient)(nil)).Elem())
 	visitClient(&internal, reflect.TypeOf((*consensus.ClientBackend)(nil)).Elem())
-	visitClientWithPrefix(&internal, reflect.TypeOf((*consensus.LightClientBackend)(nil)).Elem(), "ConsensusLight")
 	visitClientWithPrefix(&internal, reflect.TypeOf((*syncer.ReadSyncer)(nil)).Elem(), "ConsensusLightState") // this doesn't work right
```

Comment:
this needs some adjustment

Reply:
Can you explain how this should be adjusted? So this is part of the `Consensus` gRPC service.